### PR TITLE
fix: Display error if event creation fails

### LINKED ISSUE #1404: Can't Proceed After Step 3 Of Create New Event .
Current Behaviour :
   can't proceed after step 3rd of create new activity even all field is filled.
   Their is no error "Why event is not created".

Expected Behaviour :
   event is generated when user filled out all details and click on CREATE button.
   Error should be displayed like by  Toast Or TextInputLayout Error.

bug

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1404  
Error needs to displayed when user can't proceed

Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 

Now, error message would appear if user cannot proceed after step 3 of event creation.

### CODE REVIEW

File: app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java [hunk 1]
```diff
@@ -138,7 +138,10 @@ public void createEvent() {
             .subscribe(createdEvent -> {
                 onSuccess.setValue("Event Created Successfully");
                 close.setValue(null);
-            }, Logger::logError));
+            }, throwable -> {
+                Logger.logError(throwable);
+                onError.setValue(throwable.getMessage());
+            }));
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: No, handle the error and display proper message

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: Done

File: app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java [hunk 2]
```diff
@@ -138,7 +139,10 @@ public void createEvent() {
             .subscribe(createdEvent -> {
                 onSuccess.setValue("Event Created Successfully");
                 close.setValue(null);
-            }, Logger::logError));
+            }, throwable -> {
+                Logger.logError(throwable);
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: Use timber

File: app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: What about unhandled error? It should not print exceptions for unhandled errors

File: app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java (hunk 2 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: Done

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: Just for info, why timber here when the logError() function is also doing the same thing?

File: app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: Updated

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java]: It should not show toasts for unhandled errors

**Pull request merged**